internal/federationdomain/clientregistry/clientregistry_test.go

```diff
@@ -312,6 +340,43 @@ func requireEqualsPinnipedCLI(t *testing.T, c *Client) {
 		  "token_endpoint_auth_method": "none",
 		  "request_uris": null,
 		  "request_object_signing_alg": "",
-		  "token_endpoint_auth_signing_alg": "RS256"
+		  "token_endpoint_auth_signing_alg": "RS256",
+          "IDTokenLifetimeConfiguration": 0
```

Comment:
whitespace

Reply:
Oops, thanks. Will update. GoLand renders this differently, so I didn't notice that it was wrong.